.github/pull_request_template.md

```diff
@@ -0,0 +1,17 @@
+# Description
+Your description here.
+
+## Related Issues
+Closes #XXXX <- Issue number if existing
+
+## Checklist:
+<!-- Check these things before posting the pull request: -->
+- [ ]  Did you test the code?
```

Comment:
We don't want to only require only testing of changed code, but everything that was changed. Change to `Did you test your changes?`?